filecoin-proofs/src/api.rs

```diff
@@ -67,19 +65,93 @@ pub struct SealOutput {
     pub piece_inclusion_proofs: Vec<PieceInclusionProof<PedersenHasher>>,
 }
 
+#[derive(Clone, Debug)]
+pub struct GeneratePoStOutput {
```

Comment:
BLOCKING: The `verify_post` function returns a `Result<bool>` (no wrapping type). I think that the `generate_post` function should return a `Result<Vec<u8>>` instead of this struct type.

The `generate_post` function used to return a struct because I wanted to return something more descriptive than a tuple (of faults and proof-vec).